src/include/common/types/value.h

```diff
@@ -193,7 +193,8 @@ class Value {
         }
     }
 
-    std::vector<std::unique_ptr<Value>> convertKUVarListToVector(ku_list_t& list) const;
+    std::vector<std::unique_ptr<Value>> convertKUVarListToVector(
```

Comment:
Why do we need to pass in a childType here? I think we store dataType in value.

Reply:
ditto, RECURSIVE_REL doesn't explicitly store child type since it's child type is known to be INTERNAL_ID. So if want to reuse the same function, we need to pass child type as parameter